### PR TITLE
Bucket sort algorithm added

### PR DESCRIPTION
**Pull request by Anjali2001saxena**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

Bucket sort algorithm added.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: C-Plus-Plus/sort/bucketsort.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+{ 
+    float arr[] = {0.897, 0.565, 0.656, 0.1234, 0.665, 0.3434}; 
+    int n = sizeof(arr)/sizeof(arr[0]); 
+    bucketSort(arr, n); 
+  
+    cout << "Sorted array is \n"; 
+    for (int i=0; i<n; i++) 
+       cout << arr[i] << " "; 
+    return 0; 
+}
```
Comment (Kajol-Kumari) [inline C-Plus-Plus/sort/bucketsort.cpp]: Please give one extra blank line at the end of the file.

File: C-Plus-Plus/sort/bucketsort.cpp [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  
+    // Concatenate all buckets into arr[] 
+    int index = 0; 
+    for (int i = 0; i < n; i++) 
+        for (int j = 0; j < b[i].size(); j++) 
+          arr[index++] = b[i][j]; 
+}
+int main() 
+{ 
+    float arr[] = {0.897, 0.565, 0.656, 0.1234, 0.665, 0.3434};
```
Comment (Kajol-Kumari) [inline C-Plus-Plus/sort/bucketsort.cpp]: Please take the input from the user

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Anjali2001saxena Functionality is working fine!
Please add the file path in the readme's appropriate section and incorporate the suggested changes.

**Comment by Anjali2001saxena** [comment]
Mam I have made the necessary changes and update the readme file to add bucket sort. Please check.

**Review by vatsalkesarwani12 — COMMENTED** [review]
Code copied from [geeks for geeks](https://www.geeksforgeeks.org/bucket-sort-2/)

**Comment by SKAUL05** [comment]
> Code copied from [geeks for geeks](https://www.geeksforgeeks.org/bucket-sort-2/)

Closing this PR due to plagiarism.

**Pull request closed without merging**